Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
+		return "newspaperArticle";
+}
+
+function doWeb(doc, url) {
+	var arts = new Array();
+	scrape(doc);
+}
+
+function scrape(doc) {
+	var newArticle = new Zotero.Item('newspaperArticle');
+	newArticle.url = doc.location.href;
```

Comment:
don't use `doc.location.href` directly. Use the `url` that is passed into `doWeb`